exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.java

```diff
@@ -92,6 +93,23 @@ public static PrometheusHttpServerBuilder builder() {
     start();
   }
 
+  private static HttpServer createServer(String host, int port) throws IOException {
```

Comment:
I applied the recommendation from [here](https://errorprone.info/bugpattern/AddressSelection